app.rb

```diff
@@ -90,5 +91,5 @@ def verify_signature(payload_body)
 end
 
 def redmine_issue_repos
-  ENV['REDMINE_ISSUE_REQUIRED_REPOS'].to_s.split.map { |r| Regexp.new("\\A#{r}\\z") }
+  YAML.load_file('config/redmine_issue_required_repos.yaml')
```

Comment:
I wasn't clear why the regexp was initially there -- I was assuming it was due to how arrays within environment variables work. Do repository names come in truncated when long from github API? If not, I would change the code that checks for the existence to a simple `include?` instead of a regexp match.